gems/pending/appliance_console/database_replication_standby.rb

```diff
@@ -0,0 +1,81 @@
+require 'appliance_console/logging'
+require 'appliance_console/prompts'
+require 'appliance_console/database_replication'
+
+module ApplianceConsole
+  class DatabaseReplicationStandby < DatabaseReplication
+    include ApplianceConsole::Logging
+
+    REGISTER_CMD = 'repmgr standby register'.freeze
+
+    attr_accessor :appliance_pg_data
```

Comment:
Why make this an `attr_accessor`? Are we expecting it to be used external to this class? Cant we just use `PostgresAdmin.data_directory`?